### ISSUE TITLE
feat: add cap build android --debug option

### ISSUE THREAD

**Issue by distante** [ionic-team/capacitor#7047]
### Description

I am trying to implement Appium tests, for this, before running the test I need to build a debug APK with the latest code base, which currently is not possible since `cap build android` always asks for signing information

### Platform(s)

Android

### Preferred Solution

allow to run `cap build android --debug`

### Alternatives

I could try to run the `gradlew` code myself from a script...?

**Comment by matheo** [comment]
This is right what I'm looking for in 2024 to be able to send a test `apk`

**Comment by eric-horodyski** [comment]
This thread has been inactive for some time, and we’re cleaning up older items to keep the backlog focused and manageable.

Details on the effort:
[https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant today, feel free to open a new issue with current information and we’ll review it.
